pages/services/hdfs/2.5.0-2.6.0-cdh5.16.3/release-notes/index.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes
+excerpt: Release notes for v. 2.5.0-2.6.0-cdh5.16.3
+title: Release Notes
+menuWeight: 10
+model: /services/hdfs/data.yml
+render: mustache
+---
+## Version 2.5.0-2.6.0-cdh5.16.3
+
+### Feature
+- Hadoop `ipc.maximum.data.length` property is now available in service configuration.
+
+### Bug Fixes
+- Fix for JournalNodes failing to establish quorum when paxos folder is missing.
+
+### Improvements
+- Added validation to verify package names are unique and follow Universe guidelines.
+
+### Updates
+- Upgraded Hadoop distribution from cdh5.11.0 to cdh5.16.3
```

Comment:
No need to state what the previous version was, it's literally two lines down as a heading